### PR TITLE
NuGet package readme files

### PR DESCRIPTION
**Pull request by ajcvickers**
Part of #31947

This PR includes the content. I still don't know what needs to be done, if anything, to ensure the content is actually included in the packages.

### CODE REVIEW

**Comment by ErikEJ** [comment]
https://github.com/ErikEJ/EFCore.SqlServer.DateOnlyTimeOnly/blob/7ac9da8ee294e3920e7c3f27726a22156a14122f/EFCore.SqlServer.DateOnlyTimeOnly/EFCore.SqlServer.DateOnlyTimeOnly.csproj#L29C1-L32C1

https://github.com/ErikEJ/EFCore.SqlServer.DateOnlyTimeOnly/blob/7ac9da8ee294e3920e7c3f27726a22156a14122f/EFCore.SqlServer.DateOnlyTimeOnly/EFCore.SqlServer.DateOnlyTimeOnly.csproj#L8

**Comment by ajcvickers** [comment]
@ErikEJ Right, but typically the .NET build system has custom NuGet packing, so even though that's the way to do in the normal world, I suspect it not the way to do it here.

**Comment by ErikEJ** [comment]
"normal world"  😅

**Comment by AndriySvyryd** [comment]
@ajcvickers You can do what @ErikEJ suggested or you can add some code to `Directory.Build.targets` to avoid repetition, see

https://github.com/dotnet/runtime/blob/cfd909200f1c564f75ae7dcd8ac10d53949b26c2/eng/packaging.targets#L23
https://github.com/dotnet/runtime/blob/cfd909200f1c564f75ae7dcd8ac10d53949b26c2/eng/packaging.targets#L87-L89

**Comment by ajcvickers** [comment]
Thanks @AndriySvyryd. Updated to include readmes in build.

File: src/EFCore.Tools/PACKAGE.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+The Entity Framework Core tools help with design-time development tasks. They're primarily used to manage Migrations and to scaffold a `DbContext` and entity types by reverse engineering the schema of a database.
```
Comment (AndriySvyryd) [inline src/EFCore.Tools/PACKAGE.md:1]: You also need to modify `EFCore.Tools.nuspec`

**Review by AndriySvyryd — APPROVED** [review]

**Pull request merged**